### ISSUE TITLE
[Feature req] Ability to add tags to streamers

### ISSUE THREAD

**Issue by Nhoise** [freaktechnik/justintv-stream-notifications#111]
It would be nice to have the ability to add a tag to streamers which would be displayed next to their name.

Example:
> I have 3 streamers in my list which i only watch for certain events they host every so often.
> But when those events are not running i don't care about watching them.
> So it would be nice if i could add a tag of the dates on which the next event would go live to their name.

**Comment by freaktechnik** [comment]
So it's more like a "flair", as in one channel can only have one "note"?

**Comment by Nhoise** [comment]
Yes, exactly.

Another good example could be the following.
> i have someone i only watch when he's streaming his World Records attempts for Super Mario 64.
> So i would tag him with something like: "Super Mario 64 - 70 Star WR"

My list of streamers i have added is already pretty big, and sometimes i forget why i added someone.
So it could also serve as a reminder of who someone is and why i added that person.

**Comment by freaktechnik** [comment]
Todo:
 - [ ] Add tag string to database
 - [ ] Add UI for setting tag string
 - [ ] Include tag string in search

@Nhoise would it be ok if the value of the tag string is not exposed anywhere else than where you can set it? Of course you could cancel out of the tag editing thing.

**Comment by Nhoise** [comment]
Do you mean enabling/disabling the showing of the tags being included in the 'extra info in panel' checkbox in the options menu?
If so, yeah sure. That's totally fine.

If you mean actually displaying the tag next to the 'extra info' (Yellow) bit then: why not just put it next to the name? (Red)
Picture for reference.
![picture for reference](https://i.imgur.com/nwoLc4X.png)

**Comment by freaktechnik** [comment]
I'm actually thinking about even only having an about config setting that lets you disable the display of tags but make it so they are still taken into account when searching (basically it will just hide the text in the panel and channels manager).

**Comment by Nhoise** [comment]
Ah, yes. I can imagine that saving a lot of extra work.
About:config options are perfectly fine. Though it might be too 'hidden' for regular users, if you know what i mean.
I'm pretty sure a large amount of casual users have no idea that about:config even exists.

**Comment by freaktechnik** [comment]
Yes, but a large amount of users wouldn't want to add a note thing and then not display it in the panel...

**Comment by Nhoise** [comment]
Lol yeah, i guess that's true.
